testsuite/replay_verify.py

```diff
@@ -64,12 +65,26 @@
     [230_000_001, 240_000_000],
     [240_000_001, 250_000_000],
     [250_000_001, 260_000_000],
-    [260_000_001, 270_000_000],
-    [270_000_001, 278_000_000],
-    [278_000_001, sys.maxsize],
+    [260_000_001, 275_000_000],
+    [275_000_001, 285_000_000],
+    [285_000_001, sys.maxsize],
 ]
 
 
+def replay_with_retry(*args):
+    MAX_RETRIES = 5
+    attempt = 0
+    while attempt < MAX_RETRIES:
+        (n, return_code, msg) = replay_verify_partition(*args)
+        if return_code == 0:
```

Comment:
should distinguish infra error and txn_info mismatch error (don't retry on the latter)

Reply:
(test with one ver removed from the skip list)

Reply:
refactor the code to have more error types and only retry on non-txn mismatch error type.